Cargo.toml

```diff
@@ -6,15 +6,25 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-zbus = "2.1"
+zbus = {version = "2.1", features = ["gvariant"]}
 serde = "1.0"
 tokio = {version = "1.17", features = ["macros", "rt-multi-thread"]}
 futures = "0.3"
 rand = {version = "0.8", default-features = false}
+async-std = {version = "1.11", features = ["attributes"]}
```

Comment:
I would appreciate if you could drop the tokio dependency and replace the tokio::test calls with async_std::test.

Reply:
Will gladly do it. But other PR, because this one is huge already and I want to touch as little of the existing files with this one.